zerver/webhooks/sonarqube/doc.md

```diff
@@ -0,0 +1,15 @@
+Get Zulip notifications for your Sonarqube code analysis!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+   By default, the integration will use the project's name as the topic.
+
+1. On SonarQube Select **Administration** either in your project or on global level.
+   Select **Webhooks** (on global level **Configurations > Webhooks**). Click **Create**.
+
+1. Choose **Name** for webhook, set **URL** to the URL constructed above and click **Create**.
```

Comment:
Change this to:
1. Set **Name** to a name for the webhook. Set **URL** to the URL constructed above and click **Create**.

Reply:
Ok